### PR TITLE
Adds multilabel_soft_margin_loss opinfo

### PR DESCRIPTION
**Pull request by mruberry**
Per title

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -11809,6 +11824,36 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
...
+        decorators=(
+            DecorateInfo(
+                toleranceOverride({torch.float32: tol(atol=1e-4, rtol=1e-4)}),
+                "TestJit",
+                "test_variant_consistency_jit",
+            ),
+        ),
+        skips=(
+            # target doesn't require grad
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_floating_inputs_are_differentiable'),
```
Comment (ngimel) [inline torch/testing/_internal/common_methods_invocations.py:11843]: what's this? target is already created with `require_grad=False`?

**Review by ngimel — APPROVED** [review]

File: torch/testing/_internal/common_methods_invocations.py (hunk 1 above)
Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:11843]: This behavior is so rare we added a sanity check. Before it OpInfos some sample input functions were not creating every input with grad like they should

Comment (ngimel) [reply torch/testing/_internal/common_methods_invocations.py:11843]: So why is this skip needed?

**Pull request closed without merging**